shared-bindings/bitbangio/SPI.c

```diff
@@ -247,6 +279,7 @@ STATIC const mp_rom_map_elem_t bitbangio_spi_locals_dict_table[] = {
 
     { MP_ROM_QSTR(MP_QSTR_readinto), MP_ROM_PTR(&bitbangio_spi_readinto_obj) },
     { MP_ROM_QSTR(MP_QSTR_write), MP_ROM_PTR(&bitbangio_spi_write_obj) },
+	{ MP_ROM_QSTR(MP_QSTR_write_readinto), MP_ROM_PTR(&bitbangio_spi_write_readinto_obj) },
```

Comment:
Is this a tab instead of spaces? (I realize its pedantic.) Could please set your editor up to do 4 spaces when tab is hit?